flake.nix

```
@@ -0,0 +1,28 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  };
+
+  outputs = {nixpkgs, ...}: let
+    system = "x86_64-linux";
+    pkgs = nixpkgs.legacyPackages.${system};
+  in {
+    devShells.${system}.default = pkgs.mkShell {
+      packages = with pkgs; [
+        dioxus-cli
+        wasm-bindgen-cli_0_2_100
+      ];
+
+      # https://docs.rs/getrandom/0.3.3/getrandom/#webassembly-support
+      RUSTFLAGS = "--cfg getrandom_backend=\"wasm_js\"";
+
+      # With gcc (default), the WASM build fails silently
+      # at runtime there is an error message similar to https://github.com/DioxusLabs/dioxus/discussions/3807.
+      # Maybe this can be resolved with dioxus 0.7.0 due to https://github.com/DioxusLabs/dioxus/issues/4023.
```

found it! :tada:

https://github.com/briansmith/ring/blob/522afb658067bed0512a49f66bd4c07389b51ab3/mk/cargo.sh#L219-L221

and

```suggestion
      # this seems to be caused by the ring dependency: https://github.com/briansmith/ring/issues/1473
```

weird though that the issue is closed, not resolved and no follow-up is linked… ?